### ISSUE TITLE
Apply in multi-label task

### ISSUE THREAD

**Issue by oguriq** [mkocabas/focal-loss-keras#13]
Hi, I want to know if this function can be directly applied in the multi-label task. I used to use the binary-crossentropy  as my loss function in the multi-label task , now I want to use focal loss to replace it. Should I make some changes or not  ? Anybody help?

**Comment by kmh93** [comment]
I think you can not use it directly

**Comment by umbertogriffo** [comment]
@oguriq take a look at [https://github.com/umbertogriffo/focal-loss-keras](https://github.com/umbertogriffo/focal-loss-keras)

**Comment by heya5** [comment]
When I used softmax for binary classification, the program ran without error, but I deduced that the calculation process was wrong. We can see that only 0 and 1 are compared in this code, so it can't be used for multi-classification.
